Automattic/sensei
# Update packages (including select2)

### Changes proposed in this Pull Request

* Found this to be a bit easier than individually updating. I did a spike for switching to `wp-scripts` for build, but seemed quite complicated.
* I didn't update `@woocommerce/components`, `@wordpress/base-styles`, `@wordpress/components` because it seemed to need more testing.
* I did, however, upgrade `select2` and did some simple testing of a new build. 

### Testing instructions

* Run critical build and package commands.
* Ensure no regression in importer and setup wizard.
* Ensure no regression where select2 is used such as learner management.
